carta/session.py

```diff
@@ -383,6 +383,19 @@ def append_image(self, path, hdu=""):
         """
         return Image.new(self, path, hdu, True)
 
+    def open_images(self, *paths):
```

Comment:
This function will always open the default HDU for each image -- it's currently not possible to specify different HDUs. @kswang1029 will that be a problem? There are various ways that we could add this option, although they're all a bit messy. Personally I think that it's not difficult for the user to write their own loop. How often is it likely to come up?

This function will also always close any existing images. Should we also create an `append_images` function which does exactly the same thing except that it *doesn't* close existing images? This would just mean replacing the first `open_image` call with `append_image`. Instead of a separate function we could also add an option to this one.

Reply:
FWIW, the simplest and least messy way to allow optional specification of HDUs is to let the user pass in either strings or (string, string) tuples. Then in the function we'd check for this and call the underlying functions appropriately. At the cost of slightly more magic inside the function we could keep the simple API for the most common case (i.e. just plain strings for the default HDUs).